marconi-core/src/Marconi/Core/Experiment/Query.hs

```diff
@@ -109,3 +116,18 @@ instance
           dbResult <- extractDbResult
           memoryResult <- extractMemoryResult
           pure $ memoryResult <> dbResult
+
+{- | Calculates the common resume points between the resume points of multiple indexers.
+
+TODO Probably not the best place to put this.
+Ideally, it would go in `WithResume`, but that introduces a cyclic dependency when reusing that
+function in `Coordinator` and `SQLiteAggregateQuery` modules.
```

Comment:
Thoughts?

Reply:
In the `Class` module as a helper to implement `lastSyncPoints` for any indexers that handle several indexers (`Coordinator`, `SQLiteAggregateQuery`...)?